parsec/scheduling.c

```diff
@@ -175,7 +175,9 @@ int __parsec_execute( parsec_execution_stream_t* es,
                              chore_id);
 #endif
         parsec_hook_t *hook = tc->incarnations[chore_id].hook;
-        assert( NULL != hook );
+        if( NULL == hook ) {
+			goto next_chore; /* Gracefuly manage the case when NO chores are available for this task */
+		}
```

Comment:
indentation

Reply:
I had made that same change on a prior pr and got shot down for it. Let me review this why it wasn't acceptable last time around.

Reply:
Found back the discussion https://github.com/ICLDisco/parsec/pull/616#discussion_r1462462587, I could not reproduce a scenario where that would happen, and added the assert so we would find that error easily when it shows up again, which is now.